howto/install-appliance/landing.md

```diff
@@ -1,17 +1,18 @@
 The Anbox Cloud Appliance provides a deployment of Anbox Cloud to a single machine. This offering is well suited for initial prototype and small scale deployments.
-The guides in this section describe how to install the Anbox Cloud Appliance on a cloud platform.
 
-We strongly recommend that you follow the [Install the Anbox Cloud Appliance on your local machine](https://discourse.ubuntu.com/t/install-appliance/22681) tutorial before you install the appliance on a cloud platform. The tutorial guides you through installing the appliance on your local machine and makes you familiar with both the installation process and the general concepts of the Anbox Cloud Appliance.
+The guides in this section describe how to install the Anbox Cloud Appliance on different cloud platforms. There is a difference between the full Anbox Cloud installation and the Anbox Cloud Appliance (see [Variants](https://discourse.ubuntu.com/t/anbox-cloud-overview/17802#variants)). This section focuses on the **Anbox Cloud Appliance**. For instructions on how to install **Anbox Cloud**, see [How to install Anbox Cloud](https://discourse.ubuntu.com/t/install-anbox-cloud/24336).
 
-[note type="information" status="Note"]There is a difference between the full Anbox Cloud installation and the Anbox Cloud Appliance (see [Variants](https://discourse.ubuntu.com/t/anbox-cloud-overview/17802#variants)). This section focuses on the **Anbox Cloud Appliance**. For instructions on how to install **Anbox Cloud**, see [How to install Anbox Cloud](https://discourse.ubuntu.com/t/install-anbox-cloud/24336).[/note]
+We strongly recommend that you follow the [Install the Anbox Cloud Appliance on a dedicated local machine](https://discourse.ubuntu.com/t/install-appliance/22681) tutorial before you install the appliance on a cloud platform. The tutorial guides you through installing the appliance on a local machine dedicated to Anbox Cloud and makes you familiar with the installation process and the general concepts of the Anbox Cloud Appliance.
```

Comment:
Should we keep the 'local' naming scheme? As discussed in the Live review meeting, this actually applies to any kind of machine.
What do you think?

Reply:
Yes, sorry I missed that point. Keeping it with the 'local machine' term.

Reply:
Actually, wasn't this the other way around? It can be any machine, not just 'local' machines, but it should be 'dedicated' for the Appliance, as it takes over the LXD and Juju installs. What do you think?

Reply:
@ajanon Hmm, I think I misunderstood your question. 
Did you mean it should be "Install the Anbox Cloud Appliance on a dedicated machine"?

Reply:
I think so, but I'm not sure if that's how we want to present this.

Reply:
Using "dedicated" as that represents the closest meaning that we want to convey.